### DISCUSSION TITLE
Playlist display

### DISCUSSION THREAD

**Discussion by Rasb74** [rern/rAudio#440] [Fixed Issues]
From the latest release, no playlist is displayed. 

Instead of that everything is PERFECT ! 
Still great job, the best audiophile distrib ever !

**Comment by rern** [comment]
Where?

**Reply by Roland-B** [reply]
@Rasb74  Playlist has no issues for me, but then you haven't provided much info where your seeing it or which build/pi etc you are using.. for anyone, especially @rern to be able to help you.

Roland.

**Reply by Rasb74** [reply]
sorry it’s my misunderstanding, i made a confusion between adding a track for play and adding to a playlist …
no excuse for me as i’m following the raudio dev from the beginning
sorry

**Reply by Roland-B** [reply]
No problem. Great that your doing from scratch and hopefully enjoying! 

Roland.
